src/System.Xml.XmlSerializer/tests/XmlSerializerTests.cs

```diff
@@ -396,7 +396,7 @@ private static void VerifyXElementObject(XElement x1, XElement x2, bool checkFir
     public static void Xml_WithXElementWithNestedXElement()
     {
         var original = new WithXElementWithNestedXElement(true);
-        var actual = SerializeAndDeserialize<WithXElementWithNestedXElement>(original, "<?xml version=\"1.0\"?>\r\n<WithXElementWithNestedXElement xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <e1>\r\n    <ElementName1 Attribute1=\"AttributeValue1\">\r\n      <ElementName2 Attribute2=\"AttributeValue2\">Value2</ElementName2>\r\n    </ElementName1>\r\n  </e1>\r\n</WithXElementWithNestedXElement>");
+        var actual = SerializeAndDeserialize<WithXElementWithNestedXElement>(original, "<?xml version=\"1.0\"?>" + Environment.NewLine + "<WithXElementWithNestedXElement xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\">\r\n  <e1>\r\n    <ElementName1 Attribute1=\"AttributeValue1\">\r\n      <ElementName2 Attribute2=\"AttributeValue2\">Value2</ElementName2>\r\n    </ElementName1>\r\n  </e1>" + Environment.NewLine + "</WithXElementWithNestedXElement>");
```

Comment:
I have to wonder if we should use C# literal strings for these things instead of one very long line. Out of curiosity if we did use the literals would that still cause an issue? As the newlines would match that of whoever saved the file last.